orphanage_APP/resources/views/expenses.blade.php

```diff
@@ -2,8 +2,16 @@
 
 @section('content')
 <div class="home-container table-responsive-sm">
-<h4>Expenses</h4>
-<table class="table table-hover table-bordered">
+  <div class="row">
+    <div class="col-sm-10">
+      <h4>Expenses</h4>
+    </div>
+    <div class="col-sm-2">
+      <a class="btn btn-primary" href="/expenses/create">Add Expense</a>
+    </div>
+  </div>
+<br>
+  <table class="table table-hover table-bordered">
```

Comment:
@catherinekulaya I have also seen the expenses table and it is not responsive. Kindly you can apply bootstrap table responsive at https://getbootstrap.com/docs/4.1/content/tables/#responsive-tables eg.
`<div class="table-responsive">`